README.md

```diff
@@ -1,4 +1,6 @@
-# gcs-resumable-upload
+# [DEPRECATED] gcs-resumable-upload
+This repository has been deprecated. Support will end on November 1, 2023.
```

Comment:
My only suggestion would be to try and format this in red / bold just so it really stands out.